protos/class.proto

```diff
@@ -11,19 +11,21 @@ service PresenterClassService {
     rpc DeleteClass (ClassDeleteRequest) returns (Status);
 
     rpc StartClass (ClassStartRequest) returns (stream ChatMessage);
-    rpc DelegateQuizQuestion (ClassEvent) returns (Status);
+    rpc DelegateQuizQuestion (QuizQuestion) returns (Status);
+    rpc GetQuizQuestionStatistics (QuizQuestion) returns (stream QuizQuestionStatistics);
     rpc EndClass (ClassEndRequest) returns (Status);
 }
 
 service StudentClassService {
-    rpc JoinClass (JoinClassRequest) returns (stream ClassEvent);
+    rpc JoinClass (JoinClassRequest) returns (stream QuizQuestion);
     rpc SendMessageToPresenter (ChatMessage) returns (Status);
+    rpc AnswerQuizQuestion (QuizQuestionAnswer) returns (QuizQuestionResponse);
 }
 
 message Class {
     string name = 1;
     string topic = 2;
-    repeated ClassQuizQuestion classQuizQuestion = 3;
+    repeated QuizQuestionCreation quizQuestion = 3;
```

Comment:
nazwa `QuizQuestionCreation` nie jest zbyt szczęsliwa, sugeruje tworzenie czegos

czemu nie 
```
QuizQuestion {
  string uuid
  string question
  bytes (jakis obrazek?)
}
```
uczen wysyla
```
QuizAnswer {
   string questionUuid
   string answer
}
```
serwer odpowiada
```
QuizResponse {
   string status
}
```
albo uzyc tego generycznego Status

i Class mogloby zawierac liste pytan i liste QuizQuestion i liste QuizAnzwer (jaki lista wzorowych odpowiedzi) a do ucznia odsylane bedzie QuizResponse lub zwykle status

Reply:
tutaj wdarł się błąd - chodziło o po prostu QuizQuestion
QuizQuestionCreation służy do stworzenia przez prezentera pytania w systemie

Reply:
co do QuizResponse minusem twojego rozwiązania jest to, że pasuje - po pewnym czasie  - wyświetlić jaka była prawidłowa odpowiedź 
w mojej wersji jest ta prawidłowa odpowiedź gdzieś, a też da się sprawdzić status (robiłaby to apka mobilna), ale za każdą próbą odpowiedzi leciałby response z redundantnym polem ```    Option answer ```